relentless/potential/pair.py

```diff
@@ -624,14 +727,14 @@ def _energy(self, r, P, sigma_i, sigma_j, sigma_d, **params):
         r,u,s = self._zeros(r)
 
         #clamp lo
-        lo = 0.5*(sigma_i + sigma_j)
+        lo = Depletion.LowBound(sigma_i, sigma_j).value
```

Comment:
Because `sigma_i` and `sigma_j` could be design parameters, we actually need to set these variable in the coefficient dictionary directly, not in the `_energy`, `_force`, or `_derivative` methods.

The cleanest way to do this is to override the `energy`, `force`, and `derivative` methods. In these, first check if `rmin` and/or `rmax` has been set. If not, then replace the coefficient with the appropriate bound, then call the parent method:
```py
def energy(self, pair, r):
    if self.coeff[pair]['rmin'] is False:
       self.coeff[pair]['rmax'] = self.LowBound(self.coeff[pair]['sigma_i'],self.coeff[pair]['sigma_j'])
    if self.coeff[pair]['rmax'] is False:
        self.coeff[pair]['rmax'] = self.HighBound(self.coeff[pair]['sigma_i'],self.coeff[pair]['sigma_j'],self.coeff[pair]['sigma_d'])
    return super().energy(pair,r)
```

I think the only thing we need to be careful about is that we currently are not actually using a "low bound" in the sense of what we've defined in the rest of the code, since we are extrapolating the force rather than zeroing it. If this is the case, do not set `rmin`, define `lo = 0.5*(sigma_i+sigma_j)`, and we will need to be careful about computing the derivatives appropriately.